numeric.c

```diff
@@ -5739,24 +5739,56 @@ int_round(int argc, VALUE* argv, VALUE num)
 }
 
 /*
+ *  :markup: markdown
+ *
  *  call-seq:
  *    floor(ndigits = 0) -> integer
  *
- *  Returns the largest number less than or equal to +self+ with
- *  a precision of +ndigits+ decimal digits.
+ *  Returns an integer that is a "floor" value for `self`,
+ *  as specified by the given `ndigits`,
+ *  which must be an
```

Comment:
Looks good to me, but minor nit: the linebreaks on this line look unnecessarily short.

Reply:
As an old-time documentor, I've found that future edits (and the diffs thereof) go better if each clause has its own line;  that's why I broke after each of the two commas.   The link is on its own line (l. 5750) because it's 90+ characters long by itself.